toolkit-test-app/src/main/java/com/esri/arcgisruntime/toolkit/test/popup/viewmodel/PopupViewTestViewModel.kt

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright 2021 Esri
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.esri.arcgisruntime.toolkit.test.popup.viewmodel
+
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.ViewModel
+import com.esri.arcgisruntime.geometry.GeometryType
+import com.esri.arcgisruntime.layers.FeatureLayer
+import com.esri.arcgisruntime.mapping.ArcGISMap
+import com.esri.arcgisruntime.portal.Portal
+import com.esri.arcgisruntime.portal.PortalItem
+import com.google.android.material.bottomsheet.BottomSheetBehavior
+
+/**
+ * The ViewModel that provides a map, identifiableLayers in the map and the bottomsheet state.
+ */
+class PopupViewTestViewModel : ViewModel() {
```

Comment:
For simplicity, could we ommit this view model and declare all its properties directly in PopupViewTestActivity?

Reply:
@gunt0001 I have removed this ViewModel. I have taken out the property to save the bottomsheet state. 
FYI, since I have moved the `map` property from the viewModel to the activity, I cannot set the `map` on the `mapView` in the layout xml file `app:map="@{popupViewTestViewModel.map}" />` and have it show up. 
I am setting the `map` property in onCreate() `mapView.map = map`  for the map to show up in the MapView.